src/components/pages/donation_form/SubPages/DonationForm.vue

```diff
@@ -25,19 +25,29 @@
 			@set-address-type="setAddressType( $event )"
 		/>
 
-		<div class="content-card">
-			<FormSummary>
-				<template #summary-content v-if="paymentSummary">
-					<DonationSummary
-						:address="addressSummary"
-						:payment="paymentSummary"
-						:bank-data="bankDataSummary"
-						:countries="countries"
-						:salutations="salutations"
-					/>
-				</template>
-
-				<template #summary-buttons>
+		<ContentCard>
+			<template #heading v-if="paymentSummary">
+				<h2>{{ $t( 'donation_form_summary_title' ) }}</h2>
+			</template>
+			<template #content>
+				<DonationSummaryHeadline/>
+				<Summary>
+					<template #left>
+						<DonorSummarySection
+							v-if="addressSummary"
+							:address="addressSummary"
+							:countries="countries"
+							:salutations="salutations"
+						/>
+					</template>
+					<template #right>
+						<PaymentSummarySection
+							v-if="bankDataSummary"
+							:bank-data="bankDataSummary"
+						/>
+					</template>
+				</Summary>
+				<div class="switcher">
```

Comment:
@Abban Here, we are exposing a `CSS class` and that was not the idea right? Only Pattern Library new about what CSS is used behind the scenes right?

Reply:
We're going to use the real compositions as they'll help us with refactoring. You can see some of the real pattern library being imported here: `src/scss/pattern-library.scss`